src/calculator/index.ts

```diff
@@ -139,11 +141,39 @@ export default class Calculator {
     this.chain = options.chain;
   }
 
+  #votesWithCoefficientToContribution(
+    votes: VoteWithCoefficient[]
+  ): (Contribution & { recipientAddress: string })[] {
+    return votes.flatMap((vote) => {
+      const scaleFactor = 10_000;
+      const coefficient = BigInt(
+        Math.trunc((this.overrides[vote.id] ?? vote.coefficient) * scaleFactor)
+      );
+
+      const amount = BigInt(vote.amountRoundToken);
+      const multipliedAmount = (amount * coefficient) / BigInt(scaleFactor);
+
+      return [
+        {
+          contributor: vote.voter,
+          recipient: vote.applicationId,
+          recipientAddress: vote.grantAddress,
+          amount: multipliedAmount,
+        },
+      ];
+    });
+  }
+
   async calculate(): Promise<Array<AugmentedResult>> {
     const votes = await this.parseJSONFile<Vote>(
       "votes",
       `${this.chainId}/rounds/${this.roundId}/votes.json`
     );
+
+    return this._calculate(votes);
+  }
+
+  private async _calculate(votes: Vote[]): Promise<Array<AugmentedResult>> {
```

Comment:
`private` is a typescript feature, `#` is an ECMAScript spec thing. I'm generally for upholding the spec instead of relying on typescript where possible. Good discussion here: https://stackoverflow.com/questions/59641564/what-are-the-differences-between-the-private-keyword-and-private-fields-in-types.

Reply:
temp. fixed to private so as not to block this PR over style discussions.